src/index.tsx

```diff
@@ -4,7 +4,7 @@ import "./index.css";
 import App from "./App";
 import "bootstrap/dist/css/bootstrap.min.css";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+const root = ReactDOM.createRoot(document.getElementById("root")!);
```

Comment:
do you really want that negation '!' ?

Reply:
it tells ts that line 7 will always return an element and never return null. Therefore it doesn't break anything.